dirty-lib.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/actcat/sideci_go_sandbox/lib"
```

Comment:
__[gometalinter]__
* [interfacer] could not import github.com/actcat/sideci_go_sandbox/lib (cannot find package "github.com/actcat/sideci_go_sandbox/lib" in any of:
* [unconvert] could not import github.com/actcat/sideci_go_sandbox/lib (cannot find package "github.com/actcat/sideci_go_sandbox/lib" in any of:
* [gosimple] could not import github.com/actcat/sideci_go_sandbox/lib (cannot find package "github.com/actcat/sideci_go_sandbox/lib" in any of:
* [staticcheck] could not import github.com/actcat/sideci_go_sandbox/lib (cannot find package "github.com/actcat/sideci_go_sandbox/lib" in any of:

<!-- Comment by [SideCI](https://sideci.com) -->